.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,76 @@
+on:
+  push:
+    branches:
+      - main
+
+name: Deploy to AWS Production (K8s)
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      AWS_REGION: us-east-1
+      CLUSTER_NAME: maker-prod
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ env.AWS_REGION }}
+
+    - name: Login to Amazon ECR
+      id: login-ecr
+      uses: aws-actions/amazon-ecr-login@v1
+
+    - name: Extract commit hash
+      id: vars
+      shell: bash
+      run: |
+        echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+    - name: Build, tag, and push image to Amazon ECR
+      id: build-image
+      env:
+        SHA_TAG: ${{ steps.vars.outputs.sha_short }}
+        LATEST_TAG: latest
+        ENVIRONMENT_TAG: prod
+        ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+      run: |  
+        # Build Docker containers and push them to ECR cla-assistant-prod
+        docker pull $ECR_REGISTRY/cla-assistant-prod:$LATEST_TAG || true
+        docker build -t cla-assistant-prod \
+                     -t $ECR_REGISTRY/cla-assistant-prod:$SHA_TAG \
+                     -t $ECR_REGISTRY/cla-assistant-prod:$LATEST_TAG \
+                     -t $ECR_REGISTRY/cla-assistant-prod:$ENVIRONMENT_TAG \
```

Comment:
Do you want to remove "prod" from the service name? You have ENVIRONMET_TAG for that

Reply:
@eskp that is also the ECR registry name, and unfortunately ECR names are immutable. I'd prefer to not destroy all previous images.

Reply:
There are only 14 tags of ~100MB in size. You could pull them all with `-a` flag then push all up to the new repo.

If you can't be bothered I think it's also quite fine to get rid of all of them and start with a fresh repo.

Reply:
@eskp migrated all existing images from -prod to new one and changed CICD pipelines accordingly